client/components/Medical-Contact/ContactPage.vue

```diff
@@ -0,0 +1,51 @@
+<!-- Page for medical contacts management -->
+<!-- User should be authenticated in order to see this page -->
+
+<template>
+  <main>
+    <hr>
```

Comment:
I'm not a fan of the leading `<hr>`, especially since there are lines just above this in the header. If you really want it, we can talk, but I would need to adjust the app css as well (notice how the ones after a section are a bit wider but this one is not)